### DISCUSSION TITLE
Is it possible to filter sensitive data in the dashboard?

### DISCUSSION THREAD

**Discussion by amo13** [bensheldon/good_job#1087] [General]
I have jobs that get passed sensitive data as arguments, like a password and a secret key. I can prevent logging them by setting `config.active_job.log_arguments = false`, but can I also prevent them from showing up in plaintext in the jobs dashboard?

**Comment by bensheldon** [comment]
Fantastic question! There isn't, but I think that would be a really valuable feature to implement.

- I think it should respect `config.active_job.log_arguments = false` overall
- Do you think it should also have a more granular list of argument/parameter filters? (e.g. `config.filter_parameters` / `[FILTERED]`)

**Reply by amo13** [reply]
I agree with respecting `log_arguments = false`.

Granular control would certainly be an awesome feature. And yes, I guess it would be great to stick with the rails conventions and use `Rails.application.config.filter_parameters`.
